pkg/commands/artifact/run.go

```diff
@@ -600,6 +602,12 @@ func (r *runner) initScannerConfig(ctx context.Context, opts flag.Options) (Scan
 	disabledHandlers := lo.Ternary(opts.DetectionPriority == ftypes.PriorityComprehensive,
 		[]ftypes.HandlerType{ftypes.SystemFileFilteringPostHandler}, nil)
 
+	// Set the default HTTP transport
+	xhttp.SetDefaultTransport(xhttp.NewTransport(xhttp.Options{
```

Comment:
IIUC we need to set Default Transport early.
Trivy downloads trivy-db without insecure/timeout